pgsqltoolsservice/query_execution/query_execution_service.py

```diff
@@ -188,6 +190,46 @@ def on_query_complete(query_complete_params):
 
         self._start_query_execution_thread(request_context, params, worker_args)
 
+    def handle_execute_deploy_request(
```

Comment:
This method is similar to _handle_execute_query_request with the difference in what messages it sends back. Few questions:
1. Is there a reason why Deploy has to be treated differently ?
2. Could we refactor to use _handle_execute_query_request with message types ?

Reply:
Deploy is same functionally but it reports different notifications. We are reusing _start_query_execution_thread which is used in both _handle_execute_query_request  and _handle_execute_deploy_request